web/themes/custom/votegov/src/sass/uswds-overrides/usa-language-selector.scss

```
@@ -11,7 +11,6 @@
 
```

Added to the `usa-language-container` and addressed the issues

```suggestion
  background-color: #F7F9FA !important;
  border-top: 1px solid #D3D3D3;
  border-bottom: 1px solid #D3D3D3;
  @include u-padding-y(1) !important;
```

---

looks there there are some styles coming in that were overwriting the padding and bg color that is why I added the !important flag